### PR TITLE
DropDown: fix left position depending on screen width

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1067
[Problem] More menu goes out of screen
[Solution] Position of DropDown menu has to takes account
 screen size.

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
@TomaszLukawskiSam I checkted latest TAU code and can not reproduce this issue using steeps mentioned in https://github.com/Samsung/TAU/issues/1067. Did you set specific width? I can see similar issue while window resizing. Could you specify steeps to reproduce?

**Comment by grzegorz-czajkowski** [comment]
Able to reproduce this on Tizen Emulator.

File: src/js/profile/mobile/widget/DropdownMenu.js [hunk 1]
```diff
@@ -1076,7 +1084,10 @@
 					ui.elOptionWrapper.classList.add(classes.verticalMargins);
 				}
 
-				optionStyle += "top: " + offsetTop + "px; width: " + width + "px; max-height: " + height + "px;";
+				optionStyle += "top: " + offsetTop + "px; max-height: " + height + "px;";
+				if (options.inline === true) {
```
Comment (grzegorz-czajkowski) [inline src/js/profile/mobile/widget/DropdownMenu.js]: Not sure if this option is still supported and works as it should be. Are we sure that width should be excluded  one inline option is specified?

Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/DropdownMenu.js]: You are right, `inline` change is out of scope this issue.

**Review by grzegorz-czajkowski — APPROVED** [review]
Looks good to me. Verified.

**Pull request merged**